docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -691,20 +713,49 @@ To implement a persistence interceptor, complete the following steps:
 
 4.  Re-start the FHIR server.
 
-## 4.4 Resource validation
+## 4.4 Registry resources
+The IBM FHIR Server includes a dynamic registry of conformance resources.
+The registry pre-packages all [FHIR Definitions from the specification](https://www.hl7.org/fhir/R4/downloads.html)
+and uses a Java ServiceLoader to discover additional registry resource providers on the classpath.
+
+The server consults this registry for
+* StructureDefinition, ValueSet, and CodeSystem resources for resource validation.
+* SearchParameter and CompartmentDefinition resources for search.
+* ValueSet and CodeSystem resources for terminology operations.
+* And more.
+
+One common technique for extending FHIR with a set of conformance resources is to build or reference an [Implementation Guide](https://www.hl7.org/fhir/implementationguide.html).
+
+The IBM FHIR Server includes a [PackageRegistryResourceProvider](https://ibm.github.io/FHIR/guides/FHIRValidationGuide#making-profiles-available-to-the-fhir-registry-component-fhirregistry) for registeriing implementation guide resources.
```

Comment:
Misspelled "registeriing"

Reply:
My `i` key is getting sticky on this awful butterfly keyboard...I am making this typing mistake over and over...

Reply:
```suggestion
The IBM FHIR Server includes a [PackageRegistryResourceProvider](https://ibm.github.io/FHIR/guides/FHIRValidationGuide#making-profiles-available-to-the-fhir-registry-component-fhirregistry) for registering implementation guide resources.
```